### ISSUE TITLE
Tree is not fitted when using KerasnnRegressor

### ISSUE THREAD

**Issue by SheliaXin** [kaz-Anova/StackNet#31]
When I use `KerasnnRegressor` on Mac , it gives me the error:

> Exception in thread "main" java.lang.reflect.InvocationTargetException
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:497)
> 	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
> Caused by: java.lang.IllegalStateException:  Tree is not fitted
> 	at ml.Bagging.scoringhelperbagv2.<init>(scoringhelperbagv2.java:109)
> 	at ml.Bagging.BaggingRegressor.predict2d(BaggingRegressor.java:669)
> 	at ml.Bagging.BaggingRegressor.predict_proba(BaggingRegressor.java:1875)
> 	at ml.stacknet.StackNetRegressor.fit(StackNetRegressor.java:3065)
> 	at stacknetrun.runstacknet.main(runstacknet.java:522)
> 	... 5 more

Can you help me with that? Thanks!

I used:

> KerasnnRegressor loss:mean_absolute_error standardize:true use_log1p:true shuffle:true batch_normalization:true weight_init:lecun_uniform momentum:0.9 optimizer:adam use_dense:true l2:0.000001,0.000001 hidden:50,50 activation:relu,relu droupouts:0.1,0.1 epochs:20 lr:0.01 batch_size:8 stopping_rounds:10 validation_split:0.2 seed:1 verbose:false

**Comment by kaz-Anova** [comment]
Can you rerun and put `verbose:true` and print what it says?
